apps/integration/tsconfig.json

```diff
@@ -12,5 +12,11 @@
     {
       "path": "./tsconfig.editor.json"
     }
-  ]
+  ],
+  "compilerOptions": {
+    "forceConsistentCasingInFileNames": true,
+    "strict": false,
+    "noImplicitReturns": false,
+    "noFallthroughCasesInSwitch": true
+  }
```

Comment:
This change still needed?

Reply:
hm. I guess not. Not sure why I thought I needed that before...